input/metadata/properties/default.json

```
@@ -92,6 +92,20 @@
     "herd_initialization": {
       "type": "object",
       "description": "Animal generation related inputs",
+      "male_calf_rate_for_herd_init_only": {
+        "type": "number",
+        "description": "The probability of a new calf being male. This is only used for the initial herd generation and is not used for the rest of the simulation. A low value is recommended because male calves will get sold, whereas a female calf is much less likely to be removed from the herd. Two other variables that have mild effects on the count of female calves are keep_female_calf_rate and still_birth_rate.",
```

```suggestion
        "description": "The probability of a new calf being male during herd generation only. A low value is recommended because male calves will get sold, whereas a female calf is much less likely to be removed from the herd.",
```
These are useful notes about keep_female_calf_rate and still_birth_rate, but they should be in their own descriptions.